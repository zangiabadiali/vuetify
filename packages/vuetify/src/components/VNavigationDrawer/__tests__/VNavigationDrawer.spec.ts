// Components
import { VNavigationDrawer } from '..'

// Utilities
import { createTheme, VuetifyThemeSymbol } from '@/composables/theme'
import { mount } from '@vue/test-utils'
import { VuetifySymbol } from '@/framework'

describe('VNavigationDrawer', () => {
  function mountFunction (options = {}) {
    return mount(VNavigationDrawer, {
      global: {
        provide: {
          [VuetifySymbol as symbol]: { defaults: { global: {} } },
          [VuetifyThemeSymbol as symbol]: createTheme(),
        },
      },
      ...options,
    })
  }

  it('should match a snapshot', () => {
    const wrapper = mountFunction()

    expect(wrapper.html()).toMatchSnapshot()
  })
})
