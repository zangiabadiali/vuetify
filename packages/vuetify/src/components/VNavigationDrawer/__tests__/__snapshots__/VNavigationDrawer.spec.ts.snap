// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`VNavigationDrawer should match a snapshot 1`] = `
<div class="v-navigation-drawer v-theme--light">
</div>
`;
